src/index.html

```diff
@@ -15,8 +15,39 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="container">
+        <a
+          href="https://th.bing.com/th/id/OIP.axjC06-qxFXElNPCbM1MPAAAAA?rs=1&pid=ImgDetMain"
+          target="_blank"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo-moyo"
+          />
+        </a>
+        <div class="nav-bar">
+          <div class="nav-link is-active">Apple</div>
+          <div class="nav-link is-active">Samsung</div>
+          <div class="nav-link is-active">smartphones</div>
+          <div
+            class="nav-link is-active"
+            data-qa="hover"
+          >
+            laptops & computers
+          </div>
+          <div class="nav-link is-active">gadgets</div>
+          <div class="nav-link is-active">Tablets</div>
+          <div class="nav-link is-active">Photo</div>
+          <div class="nav-link is-active">Video</div>
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'is-active' suggests a state style rather than content meaning. Also, the navigation should use semantic tags like `nav` with `ul` and `li` elements for navigation items, with `a` tags for the links.